preflight-checks/nccl-pairwise/nccl_pairwise/protos/crd_pb2.py

```
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Ruff (0.15.12)

[warning] UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)